COMMUNICATION.md

```diff
@@ -0,0 +1,21 @@
+# Communication within the OpenTelemetry Project
+
+## Document purpose
+
+OpenTelemetry has a [Code of Conduct](https://github.com/open-telemetry/community/blob/master/code-of-conduct.md) which primarily pertains to "bannable" offenses: things like harassment, personal attacks, and so on. Certainly OpenTelemetry does not tolerate any such behavior.
+
+And while we all deserve to collaborate without fear of overt harassment and personal attacks, that is a very low bar! This document is more aspirational and has fewer "teeth" (i.e., it's not about "banning" people), but it's also very important: it describes **how we wish to communicate with each other** and **how we aspire to create an environment that cultivates trust**, even in the face of inevitable technical and organizational disagreement.
+
+Finally, this is **a living document** and should be adapted over time as the OpenTelemetry project evolves. At any time, there are certainly errors of omissions or even simple clarifications we can make to the wording here, and any and all are welcome to propose such changes.
+
+Now, without further ado:
+
+## Communication guidelines
+
+What follows are an unordered list of principles that guide our communication with each other – both active community members as well as those who may just be "passing through" to file an issue or suggest an improvement.
+
+* **Critique ideas, not people:** It's fine and expected to disagree about technology and project direction. Even when those disagreements are significant or frustrating, though, it's not fine to criticize the participants. (In fact, it's usually a sign that there's no strong technical argument being made)
```

Comment:
I feel like this is a good place to reference this blog post: https://developers.redhat.com/blog/2019/07/08/10-tips-for-reviewing-code-you-dont-like/ . I know your point isn't about code reviews, but most of the arguments in that blog post apply to general communication as well.

Reply:
Good point, I have added a link. I looked around for other articles, but couldn't find one that's better.